dev/NavigationView/NavigationView.cpp

```diff
@@ -1115,6 +1119,51 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    if (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Top
```

Comment:
DisplayMode in PaneDisplayMode=Top is always minimal no? I think just checking whether DisplayMode = minimal should be sufficient.

Reply:
Yep, checking for DisplayMode = minimal is enough here. Fixed